app/controllers/evaluations_controller.rb

```diff
@@ -28,6 +29,55 @@ def process_feedback
     end
   end
 
+  def export
+    if params[:provider].present?
+      mooc_provider = MoocProvider.find_by(name: params[:provider])
+      if mooc_provider.present?
+        courses = Course.where(mooc_provider: mooc_provider)
+      else # given provider not valid
+        courses = Course.all
+      end
+    else # no provider given
+      courses = Course.all
+    end
+    @courses_with_evaluations = []
+    courses.each do |course|
+      course_evaluations = Set.new
+      course.evaluations.each do |evaluation|
+        evaluation_object = {
+          rating: evaluation.rating,
+          description: evaluation.description,
+          creation_date: evaluation.created_at,
+          total_feedback_count: evaluation.total_feedback_count,
+          helpful_feedback_count: evaluation.positive_feedback_count,
+          course_status: evaluation.course_status.to_sym,
+          is_verified: evaluation.is_verified
+        }
+
+        if evaluation.rated_anonymously
+          evaluation_object[:user_name] = t('evaluations.anonymous')
+        else
+          evaluation_object[:user_name] = "#{evaluation.user.first_name} #{evaluation.user.last_name}"
+        end
+        course_evaluations << evaluation_object
+      end
+
+      course_with_evaluations = {
+        course_id_from_provider: course.provider_course_id,
+        mooc_provider: course.mooc_provider.name,
+        overall_rating: course.calculated_rating,
+        number_of_evaluations: course.rating_count,
+        evaluations: course_evaluations
+      }
+
+      @courses_with_evaluations.push course_with_evaluations
+    end
+
+    respond_to do |format|
+      format.json { render :export }
```

Comment:
Rails will throw an error (which is usually not good), if you try to reach the page without the '.json' suffix.